### ISSUE TITLE
Assign To My Team Endpoint - adjust to allow multiple teams per project

### ISSUE THREAD

**Issue by Peyton-McKee** [Northeastern-Electric-Racing/FinishLine#1171]
### Description

we want to allow a project to be assigned to multiple teams. Currenlty our setProjectTeam endpoint overwrites the current projects team with the given one, instead we want to add it to projects Team[] array from #1170 

Also if a project is currently a assigned to a team, it should be unassigned when this endpoint is hit (the frontend 'assign to my team' button under the actions menu on the project page should be updated to say 'unassign from my team' if the team has the project assigned to them) blocked by #1170 

### Acceptance Criteria

Doesnt overwrite but instead adds the team to the projects Team[] array. 

### Proposed Solution

^

**Comment by RChandler234** [comment]
Already done!
